storm-core/src/jvm/org/apache/storm/command/UploadCredentials.java

```diff
@@ -111,7 +112,13 @@ public static void main(String[] args) throws Exception {
         // use the local setting for the login config rather than the topology's
         topologyConf.remove("java.security.auth.login.config");
 
-        StormSubmitter.pushCredentials(topologyName, topologyConf, credentialsMap, (String) cl.get("u"));
+        boolean throwExceptionForEmptyCreds = (boolean) cl.getOrDefault("e", false);
```

Comment:
No need for default value here. above `boolOpt` is already default to `false`

Reply:
Fixed